submissions.erb

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Submissions</title>
+    <link rel="stylesheet" href="home.css">
```

Comment:
there is no css file, i was just using it as a placeholder but it has no purpose